Jenkinsfile

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+import co.elastic.matrix.*
+import groovy.transform.Field
+
+/**
+This is the parallel tasks generator,
+it is need as field to store the results of the tests.
+*/
+@Field def rumTasksGen
+
+pipeline {
+  agent { label 'linux && immutable' }
+  environment {
+    BASE_DIR="src/github.com/elastic/apm-agent-js-base"
+    NOTIFY_TO = credentials('notify-to')
+    JOB_GCS_BUCKET = credentials('gcs-bucket')
+    PIPELINE_LOG_LEVEL='INFO'
+    CODECOV_SECRET = 'secret/apm-team/ci/apm-agent-rum-codecov'
```

Comment:
can we start using codecov on all PR's?

Reply:
if we have test output files, we could make it

Reply:
Nice, will add the script necessary to run the codecov and will ask help if required.